src/app/shared/context-resolver.service.ts

```diff
@@ -17,7 +17,11 @@ export class ContextResolver implements Resolve<Context> {
 
   constructor(private contextService: ContextService, private router: Router) {
     // The default place to navigate to if the context cannot be resolved is /home
-    this._lastRoute = '/_home';
+    this._lastRoute = '/_home'; // '/_error';
```

Comment:
@joshuawilson we can also redirect to `_error` page. in that case, we might need to revisit the UX of the error page.